rust/sealevel/environments/mainnet3/warp-routes/eclipseusdc/token-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "solanamainnet": {
+    "type": "collateral",
+    "decimals": 6,
+    "interchainGasPaymaster": "AkeHBbE5JkwVppujCQQ6WuxsVsJtruBAjUo6fDCFp6fF",
+    "token": "EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v",
+    "splTokenProgram": "token"
+  },
+  "eclipsemainnet": {
+    "type": "synthetic",
+    "decimals": 6,
```

Comment:
hoping that 6 here works, and the only constraint is that the upper limit is 9 decimals @tkporter?

Reply:
yeah agree 6 works best given both Solana and Ethereum use it. and yes afaik 9 as an upper limit is the only constraint